pkg/security/scc/byrestrictions.go

```diff
@@ -31,20 +45,20 @@ func pointValue(constraint *securityapi.SecurityContextConstraints) int {
 	// strategies in order of least restrictive to most restrictive
 	switch constraint.SELinuxContext.Type {
 	case securityapi.SELinuxStrategyRunAsAny:
-		points += 4
+		points += runAsAnyUserWeight
```

Comment:
The only way that I know is to use Rust language where `match` that doesn't cover all `enum` values won't compile.